libs/libarchfpga/src/physical_types.h

```diff
@@ -763,6 +799,16 @@ struct t_physical_tile_port {
  *
  * index: Keep track of type in array for easy access
  * physical_tile_index: index of the corresponding physical tile type
+ *
+ * A logical block is the implementation of a component'ss functionality of the FPGA device
+ * and it identifies its logical behaviour and internal connections.
+ *
+ * The logical block type is mainly used during the packing stage of VPR and is used to generate
+ * the packed netlist and all the corresponding blocks and their internal structure.
+ *
+ * The logical blocks than get assigned to a possible physical tile for the placement step.
+ *
+ * A logical block must correspond to at least one physical tile.
  */
 struct t_logical_block_type {
     char* name = nullptr;
```

Comment:
explain equivalent_tiles (below):  comment that it is a lit of the physical tiles at which one could place this type of netlist block?

Reply:
Done